openrpc/src/methods/getTransactions.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "getTransactions",
+  "summary": "returns a list of transactions with their details",
+  "description": "The getTransactions method returns details of a list of transactions starting from the user specified start ledger or cursor. The soroban-rpc system maintains a restricted history of recently processed transactions, with the default retention window set at 1440 ledgers, approximately equivalent to a 2-hour timeframe. For private soroban-rpc instances, it is possible to modify the retention window value by adjusting the transaction-retention-window configuration setting. Due to this limited history, clients should make sure that the requests for transactions fall within the ledger range stored by their rpc instance.",
```

Comment:
We are going to maintain a history of 7 days now.

Reply:
We can only do that once the work to back events by a DB is complete, because otherwise you will have different views of history in memory vs in the database.

Reply:
@janewang Yeah the 7 days window only goes into effect once all db changes are complete.